src/i18n/ca/index.js

```diff
@@ -180,11 +180,11 @@ export default {
       headerName: 'Comparteix',
       title: 'Comparteix',
       help: 'Comparteixi el mapa actual amb les geometries que hagi dibuixat.',
-
```

Comment:
@aroiginfraplan please keep this line as separator of mandatory entries from optional ones